### PR TITLE
Normalize and enforce license disclaimer headers

### PR DESCRIPTION
**Pull request by apmorton**

### CODE REVIEW

File: lib/compilers/fortran.js [hunk 1]
```diff
@@ -1,5 +1,4 @@
-// Copyright (c) 2018, Forschungzentrum Juelich GmbH
-//                     Juelich Supercomputing Centre
+// Copyright (c) 2018, Forschungzentrum Juelich GmbH, Juelich Supercomputing Centre
```
Comment (apmorton) [inline lib/compilers/fortran.js:1]: non trivial change

File: lib/compilers/ispc.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017-2020, Matt Godbolt & Rubén Rincón
+// Copyright (c) 2017, Matt Godbolt & Rubén Rincón
```
Comment (apmorton) [inline lib/compilers/ispc.js:1]: non trivial change

File: static/modes/fortran-mode.js [hunk 3]
```diff
@@ -1,32 +1,30 @@
-//    Copyright (c) 2015, Thomas E. Dunn
```
Comment (apmorton) [inline static/modes/fortran-mode.js:1]: non trivial change

File: static/modes/gccdump-rtl-gimple-mode.js [hunk 4]
```diff
@@ -1,6 +1,3 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (apmorton) [inline static/modes/gccdump-rtl-gimple-mode.js:1]: non trivial change

**Comment by apmorton** [comment]
I have marked non-trivial changes (whitespace, punctuation, addition of license in case of missing) with review comments above - I am unsure how to handle these, or if the changes I made were acceptable.

**Review by mattgodbolt — APPROVED** [review]
They look perfectly reasonable to me.

**Review by partouf — APPROVED** [review]
Non-trivial changes look to be Ok, there's no loss of information (unlike the git history apparently because of some file moves)

**Review by RabsRincon — APPROVED** [review]

**Pull request merged**